rackcity/views/user_views.py

```diff
@@ -89,7 +91,7 @@ def netid_login(request):
 
 
 @api_view(['POST'])
-@permission_classes([IsAdminUser])
+@permission_required(PermissionPath.USER_WRITE.value, raise_exception=True)
```

Comment:
Do you need USER_WRITE permission to view users?

Reply:
oh good catch. technically not write permission, but only admin users can view other users, and only admin users have user write permission, so I think I will keep this;

Reply:
Okay. @juliasaveliff